libraries/resource_abstractor_client/client_helper.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import os
+
+from requests import exceptions
+
+RESOURCE_ABSTRACTOR_ADDR = (
+    f"http://{os.environ.get('RESOURCE_ABSTRACTOR_URL')}:"
+    f"{os.environ.get('RESOURCE_ABSTRACTOR_PORT')}"
+)
+
+
+def make_request(method, api, **kwargs):
+    url = f"{RESOURCE_ABSTRACTOR_ADDR}{api}"
+    try:
+        response = method(url, **kwargs)
+        response.raise_for_status()
+        return response.json()
+    except exceptions.RequestException:
+        logging.warning(f"Calling {url} not successful.")
+
+    return None
```

Comment:
Why does it matter if None is returned at the end of the function or at the except?

Reply:
Practically it does not make any difference, so this is a Nitpick.
It is a matter of personal taste I guess 😅

E.g. It reads to me similar to this:
```
If <some_condition>:
   <do smth>
   return A
else:
  <do smth else>

return B
```

Instead of
```
If <some_condition>:
   <do smth>
   return A
else:
  <do smth else>
  return B
```

So feel free to ignore and resolve this comment 🙂

Reply:
I prefer the first scenario to be honest, because it indicates that there is a default return - generally speaking. 
This is a nitpick as well but as a matter of taste I prefer to have less indented code. eg in the first scenario return B is less indented than the other scenario.

Reply:
How about this? Would you prefer it more?

```py
def make_request(method, api, **kwargs):
    url = f"{RESOURCE_ABSTRACTOR_ADDR}{api}"

    result = None
    try:
        response = method(url, **kwargs)
        response.raise_for_status()
        result = response.json()
    except exceptions.RequestException:
        logging.warning(f"Calling {url} not successful.")

    return result
```

Reply:
Hmm. To be fair this is such a tiny style detail + it is your component so feel free to use what feels right for you.
We both can use the time in other ways I assume 🙂
Sorry for the disturbance with my first comment